site/src/pages/UsersPage/storybookData/groups.ts

```diff
@@ -0,0 +1,546 @@
+import type { Group, GroupSource, User } from "api/typesGenerated";
+import { MockUsers } from "./users";
+
+function findMockedUserById(id: string): User | undefined {
+	const user = MockUsers.find((user) => user.id === id);
+	return user;
+}
+
+// These values were retrieved from the Coder API. Sensitive information such as
+// usernames, names, and emails has been replaced with fake user data to protect
+// privacy. This user data comes from the MockUsers to keep consistency.
+export const MockGroups: Group[] = [
```

Comment:
Should we replace/remove the ones with external avatar URLs?  Could be weird if our tests are making requests for these external jpgs and such